rpm/cloud-info-provider-openstack.spec

```diff
@@ -4,7 +4,7 @@
 
 Summary: Cloud Information provider meta-package for OpenStack
 Name: cloud-info-provider-openstack
-Version: 0.9.1
+Version: 0.10.0
```

Comment:
if we don't have a hard dependency on the info-provider version, do we really need to bump the meta package version? I guess it's harmless but it's a bit strange to have as only change the version.

Reply:
Yes, I don't like it much, but thought it would be easier/clearer to have everything at the same level, having 3 different versions for something that is always released as once could be strange. And also the github tag would only be for the cloud-info-provider package and not for the meta pacakges.